### ISSUE TITLE
Getting Started With Caliburn.Micro

### ISSUE THREAD

**Issue by CrossSlide** [MarimerLLC/cslaforum#490]
I want to use Caliburn.Micro on a WPF project. I'm new to MVVM and have only studied the Bxf framework in book 4. 

I'm also looing to use CslaContrib-Caliburn.Micro.V2.WPF. It say it's for v2 of Caliburn.Micro. Do I need to find the Caliburn.Micro v2 or can I use the latest Caliburn.Micro v3.2?

**Comment by tfreitasleal** [comment]
Good point. I guess the partt of Caliburn.Micro we need didn't change, but I'm not sure. Please try and report back.

**Comment by CrossSlide** [comment]
Thanks @tfreitasleal, I'll try and report back but it may be a while. It's all new to me...

**Comment by CrossSlide** [comment]
Hi @tfreitasleal,
Is there any sample code out there on how to use  CslaContrib-Caliburn.Micro.V2.WPF?

**Comment by tfreitasleal** [comment]
None that I knwo of. I have an incomplete sample I can send you. Email me tfreitasleal  gmail so I can zip it to you.

**Comment by Chicagoan2016** [comment]
@CrossSlide , If you get some down time, please share your experience with us, I had looked at Prism in the past but didn't actually integrate it with CSLA.
Thanks

**Comment by CrossSlide** [comment]
@Chicagoan2016, So far I really like it. @tfreitasleal has also been a big help. I don't have a truly function app yet but I'm on my way. I have not found any problems with Caliburn.Micro v3.2 and CslaContrib-Caliburn.Micro.V2.WPF working together so far. Overall I'm please with the combination.

**Comment by tonydrake** [comment]
Agree - please share (a blog erc) - I spent time with Caliburn Micro, and
MVVMCross, but found MVVMLight the only one what worked for me... and I had
to work around some oddities in CSLA to make the experience real for users
(autosave on data change, and I still get inconsistent error flagging).
There is an issue with MVVMCross where you cannot re-validate a view (it
shows with calling any event you can hook into) before you show to a user
(views are singletons)  so you can't load a fresh dataset in a detail after
the initial create) which killed it for me, but CM never wanted to play at
all with valid/dirty objects and error management.
I would love to get CM working, but xamerin/aspnet core is more important
to me now...

**Comment by rockfordlhotka** [comment]
@tonydrake you might find that version 4.6 helps, because it supports in-place editing of an object graph. Or more accurately, it supports merging the result of a `Save` back into the original graph, rather than rebinding the UI to a new object graph.
